lib/index.ts

```diff
@@ -67,5 +69,11 @@ export function shopifyApi<T extends ShopifyRestResources>(
     }) as T;
   }
 
+  shopify.logger
+    .debug(
```

Comment:
I was thinking, this is a fairly useful thing to log as it also serves the purpose of letting the app know we're ready for it. Should this be an info log since it only happens once?